db/migrate/20161102212911_add_list_data_to_disability_claim.rb

```diff
@@ -0,0 +1,5 @@
+class AddListDataToDisabilityClaim < ActiveRecord::Migration
+  def change
+    add_column :disability_claims, :list_data, :json, null: false, default: {}
```

Comment:
I only needed a default because the column is non-null, so I couldn't upgrade the DB without a default. I don't think it's necessary for the old claims. We could add a second migration to remove the default, if it's cleaner to keep the two columns the same.